vue/components/forms/personalization-form/reference.vue

```diff
@@ -111,7 +111,7 @@ export const Reference = {
          */
         tabProperties: {
             type: Array,
-            default: () => ["font", "style"]
+            default: () => ["font", "font_size", "style", "position", "color"]

```

Comment:
On top of this, I guess we should omit properties for which we have a single value. For instance, if `position` only has one possible value then there is no reason why it should go in the tab message. For that you can use `this.properties` in `getTabMessage`.